package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@time-loop/md-to-quill-delta",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Since it is a bugfix, version should be 1.1.1 to follow semver rules. 
```suggestion
  "version": "1.1.1",
```

---

I was thinking about it, but thought this was better.

I guess technically if something had a 'break' before, it was being ignored so it added new functionality that might not just be as small as a simple bug fix. I felt it better matched:

> MINOR version when you add functionality in a backward compatible manner

In this case I already updated the FE and it isn't a huge deal, so just going to leave as is for now unless you feel strongly.

Going to make sure we do a regression on this to make sure no unintended side effects.